Makefile

```diff
@@ -20,6 +26,15 @@ CNI_SPEC_VERSION?=0.3.1
 # Ensure that the dist directory is always created
 MAKE_SURE_DIST_EXIST := $(shell mkdir -p dist)
 CALICO_BUILD?=calico/go-build
+ETCD_IMG?=quay.io/coreos/etcd
+
+# Set docker image for arm64
+ifeq ($(ARCH), arm64)
+CALICO_BUILD=linaro/ci-aarch64-leg-go-build-alpine
```

Comment:
Would you be able to submit a Dockerfile [here](https://github.com/projectcalico/go-build)? Then we could have official calico/go-build releases. 

This is also what we did for [ppc64](https://github.com/projectcalico/go-build/blob/master/Dockerfile.ppc64le)

Reply:
@caseydavenport 
No problem.  I would submit it ASAP.  
And since my company requires an internal review before any submit, I will submit it after that.